Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1839,10 +1839,23 @@
     io_registry_get_root_entry
     io_service_close))
 
+#if HAVE(MACH_HARDENED_EXCEPTIONS)
+        (with-filter (require-not (lockdown-mode))
+            (allow mach-message-send (kernel-mig-routine task_register_hardened_exception_handler)))
+            (allow mach-message-send (kernel-mig-routine thread_adopt_exception_handler)))
+#else
+        (with-filter (require-not (lockdown-mode))
+            (allow mach-message-send (kernel-mig-routine thread_set_exception_ports)))
+#end
+
 (define (kernel-mig-routines-blocked-in-lockdown-mode-avoid-telemetry) (kernel-mig-routine
     thread_info
 #if !ENABLE(BLOCK_SET_EXCEPTION_PORTS)
     thread_set_exception_ports
+#if HAVE(MACH_HARDENED_EXCEPTIONS)
+    task_register_hardened_exception_handler
+    thread_adopt_exception_handler
+#endif
```

Comment:
Should `thread_set_exception_ports` only appear in the `#else` of `HAVE(MACH_HARDENED_EXCEPTIONS)`?